apps/prairielearn/python/prairielearn/question_utils.py

```
@@ -213,7 +215,19 @@ def add_submitted_file(
         >>> add_submitted_file(data, "foo.txt", "base64-contents", mimetype="text/plain")
         >>> data["submitted_answers"]
         {"_files": [{"name": "foo.txt", "contents": "base64-contents", "mimetype": "text/plain"}]}
+        >>> add_submitted_file(data, "bar.txt", raw_contents="raw contents")
+        >>> data["submitted_answers"]
+        {"_files": [{"name": "foo.txt", "contents": "base64-contents", "mimetype": "text/plain"},
+                    {"name": "bar.txt", "contents": "cmF3IGNvbnRlbnRz"}]}
     """
+    if base64_contents is None:
```

Can we throw an error if both are None?